identity/peer.go

```diff
@@ -0,0 +1,30 @@
+package identity
+
+import (
+	"fmt"
+
+	"perun.network/go-perun/wallet"
+)
+
+type NetworkAdapter string
+
+const (
+	TCPIP NetworkAdapter = "tcpip"
+)
+
+type URL struct {
+	IPAddr   string
+	Port     uint16
+	Endpoint string
+}
+
+func (u URL) Address() string {
+	return fmt.Sprintf("%s:%d", u.IPAddr, u.Port)
+}
+
+type Peer struct {
+	Alias          string
+	PerunID        wallet.Account
```

Comment:
The `PerunID` is defined as `peer.Address`.

Reply:
Updated.